packages/integrations/components/integrationsManager/index.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// User Config.
+import userConfig from '@irvingjs/irving.config';
+// Redux.
+import { connect } from 'react-redux';
+import { actionHydrateComponents } from '../../actions';
+// Integration components.
+import GoogleAnalytics from '../googleAnalytics';
```

Comment:
Can we abstract this part into another file? It'd be nice to manage the integrations component map from a different location.